project/scalatest.scala

```diff
@@ -115,7 +115,7 @@ object ScalatestBuild extends Build {
   def sharedSettings: Seq[Setting[_]] = Seq(
     javaHome := getJavaHome(scalaBinaryVersion.value),
     scalaVersion := buildScalaVersion,
-    crossScalaVersions := Seq(buildScalaVersion, "2.10.6", "2.12.0"),
+    crossScalaVersions := Seq(buildScalaVersion, "2.10.6", "2.11.12"),
     version := releaseVersion,
```

Comment:
did you mean to drop 2.12 here?

Reply:
Nope, 2.12 is what `buildScalaVersion` is on line 37